src/SourceGenerators/Uno.UI.Tasks/Content/uno.ui.tasks.assets.targets

```diff
@@ -198,7 +198,7 @@
 	as nuget packages content.
 	-->
 	<Target Name="_UnoAssetsGetCopyToPublishDirectory"
-	BeforeTargets="GetCopyToPublishDirectoryItems">
+			BeforeTargets="GetCopyToPublishDirectoryItems;_UnoGeneratePriMarker">
```

Comment:
Just for personal knowledge: what's the purpose/benefit of this change?

Reply:
@ajpinedam Trying to unblock toolkit build.

It fails with `Error NU5019: File not found: 'D:\a\1\s\src\library\Uno.Toolkit.Cupertino\bin\Uno.Toolkit.UI.Cupertino\Release\net7.0-ios\Uno.Toolkit.UI.Cupertino.uprimarker'.`

What happens there is that `_UnoGeneratePriMarker` was run *after* `_UnoAssetsGetCopyToPublishDirectory`, and so uprimarker files wasn't copied.

I'm not yet sure what is special about the toolkit build though.